### DISCUSSION TITLE
Tried every proof I have, contacted GitHub support, all failed.

### DISCUSSION THREAD

**Discussion by tobedecided0601** [community/community#18664] [GitHub Education]
Recently I tried to apply for the GitHub Student Pack but I got rejected almost instantly and the reason said that the document has been modified (absolutely not). I've been rejected for 5 times in a row. Also my university don't provide any email so documents is what I could do for now.

I even tried contacting GitHub Support but no one replies back.

What should I do now? Or should I just give up getting the pack?

**Comment by AndyAn7** [comment]
dont give up; you must believe. it will come through for you...now, try again.

**Comment by sharklayer** [comment]
Actually no one will help you, if any maintainer answer you here it will be always the same scripted answer. If github support replies you back it will be always the same answer doesn't matter what you say.
My conclusion after trying 32 requests in less than a month is that they don't care and they don't want to help you, they just want to make you beg until they get bored.

**Comment by tobedecided0601** [comment]
Guess I will keep submitting the same proof since I don't have anything else, lol

**Comment by Black0wI** [comment]
That's the same for me. As the school admin & Github Campus Advisor, i'm quite stuck ...

**Comment by NightStrang6r** [comment]
The same problem...

**Comment by tobedecided0601** [comment]
Just got a school issued email and I was be able to grab the pack.
